ducc/docker-api/util.go

```diff
@@ -44,7 +44,7 @@ var thinImageVersion = "1.0"
 
 // m is the manifest of the original image
 // repoLocation is where inside the repo we saved the several layers
-// origin is an ecoding fo the original referencese and original registry
+// origin is an encoding of the original references and original registry
```

Comment:
My tooling defaults to ignoring 2 character words. So the `fo` change is a manual one-off.